packages/cli/src/commands/server/middleware/getSecurityHeadersMiddleware.ts

```diff
@@ -17,7 +17,7 @@ export default function getSecurityHeadersMiddleware(
   // Block any cross origin request.
   if (
     req.headers.origin &&
-    req.headers.origin !== `http://localhost:${address.port}`
```

Comment:
We should be able to remove the address variable then?

Reply:
good catch! Not sure why I still try to fix things without checking out repo's😅